plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -123,6 +124,12 @@ export const MembersListCard = (_props: {
   const { loading, error, value: members } = useAsync(async () => {
     const membersList = await catalogApi.getEntities({
       filter: { kind: 'User' },
+      where: {
+        [RELATION_MEMBER_OF]: {
```

Comment:
What's the difference between this and

```
filter: {
  [RELATION_MEMBER_OF]: stringifyEntityRef({ kind: 'group', name: groupName})
}
```
?

Reply:
Good question, nothing I guess. I was thinking the `fields` and `filter` was more of a "select {fields} from {filter}" although that's not really an accurate abstraction. Perhaps it makes sense for arbitrary properties in `filter` instead. I do realise now that this can be quite confusing.

Reply:
more like `select <fields> from entities where <filter>`